components/x-gift-article/src/HighlightRecipientMessage.jsx

```diff
@@ -19,6 +19,7 @@ export default ({ handleSaveAnnotations, handleCloseRecipientMessage }) => {
 			<button
 				className="x-gift-article__highlight-recipient-message__close"
 				type="button"
+				content="Close"
```

Comment:
**Change Request:** This isn't what I expected when I asked for the button to have text and as far as I am aware the `content` attribute isn't valid on a `button` element.

Please add the text to the button element eg `<button>Close</button>` and then use CSS to visually hide that text so sighted users can't see it but its still accessible to screen readers / assistive tech. We have a helper class for doing this called `o-normalise-visually-hidden` you can see it used [here](https://github.com/Financial-Times/x-dash/blob/4926cfbbf8de7b83615b1dd72413292cf29d3b48/components/x-teaser/src/Title.jsx#LL32C23-L32C50)